packages/ui/src/lib/utils/url.ts

```diff
@@ -0,0 +1,13 @@
+export function determinePlatform(userAgent: string) {
+	const ua = userAgent.toLowerCase();
+
+	if (ua.includes('mac os')) {
+		return 'macos';
+	} else if (ua.includes('windows')) {
+		return 'windows';
+	} else if (ua.includes('linux')) {
+		return 'linux';
+	}
+
+	return 'unknown';
+}
```

Comment:
This file looks incorrectly named. Let's change it to platform.ts?

Reply:
Yeah the thinking was that we're getting this out of the request headers, but `url` isn't even a good name for that either. Changed.